### ISSUE TITLE
acpi_power_button breaks sysfs_tablet_mode

### ISSUE THREAD

**Issue by hacker1024** [sebanc/brunch#398]
As the title says, when the former option is enabled, writing to the sysfs files enabled by the latter has no effect. `force_tablet_mode` is broken too.

**Comment by sebanc** [comment]
I tried to reproduce this issue but could not, I am currently running the below release, could you try it ? (validate the github invite first)
https://github.com/sebanc/brunch-testing/releases

**Comment by rayqusa** [comment]
yeah, it break. my laptop is thinkpad yoga 260 6200u. if I add sysfs_tablet_mode, it wont booting and repairing chrome OS. then unable to boot until reinstall it

**Comment by hacker1024** [comment]
Sorry for the delay, I'll try again with the most up-to-date build.
